test/jellyfish_web/controllers/room_controller_test.exs

```diff
@@ -8,42 +8,44 @@ defmodule JellyfishWeb.RoomControllerTest do
   describe "index" do
     test "lists all rooms", %{conn: conn} do
       conn = get(conn, Routes.room_path(conn, :index))
-      assert json_response(conn, 200)["data"] == []
+      assert json_response(conn, :ok)["data"] == []
     end
   end
 
   describe "create room" do
     test "renders room when data is valid", %{conn: conn} do
-      conn = post(conn, Routes.room_path(conn, :create), max_peers: 10)
-      assert %{"id" => id} = json_response(conn, 201)["data"]
+      conn = post(conn, Routes.room_path(conn, :create), maxPeers: 10)
+      assert %{"id" => id} = json_response(conn, :created)["data"]
 
       conn = get(conn, Routes.room_path(conn, :show, id))
 
       assert %{
                "id" => ^id,
-               "config" => %{"max_peers" => 10},
+               "config" => %{"maxPeers" => 10},
```

Comment:
Why do you use camel case?

Reply:
Doesn't JSON use camelCase by convention? Also, that how it was specified [here](https://github.com/jellyfish-dev/jellyfish/pull/3/files#diff-f26912d4d2c7811bd456bf4f854b781a6a26384ea258dc55dfb592953cc4f9faR141).

Reply:
Yeah you are right.